exercises/practice/hamming/hamming_test.go

```diff
@@ -4,39 +4,30 @@ import "testing"
 
 func TestHamming(t *testing.T) {
 	for _, tc := range testCases {
-		got, err := Distance(tc.s1, tc.s2)
-		if tc.expectError {
-			// check if err is of error type
-			var _ error = err
-
-			// we expect error
-			if err == nil {
-				t.Fatalf("Distance(%q, %q); expected error, got nil.",
-					tc.s1, tc.s2)
-			}
-		} else {
-			// we do not expect error
-			if err != nil {
-				t.Fatalf("Distance(%q, %q) returned unexpected error: %v",
-					tc.s1, tc.s2, err)
-			}
-			if got != tc.want {
-				t.Fatalf("Distance(%q, %q) = %d, want %d.",
-					tc.s1, tc.s2, got, tc.want)
+		t.Run(tc.description, func(t *testing.T) {
+			got, err := Distance(tc.s1, tc.s2)
+			if tc.expectError {
+				if err == nil {
+					t.Errorf("Distance(%q, %q) expected error, got: %d", tc.s1, tc.s2, got)
```

Comment:
should say `got nil`, the value of "got" is not relevant in the message

Reply:
We leave it like that as discussed in Slack